scripts/install/frontend

```diff
@@ -11,7 +11,3 @@ rm -rf ./custom_components/hacs/hacs_frontend_experimental
 scripts/install/pip_packages --target=./custom_components/hacs "hacs_frontend==${EXPERIMENTAL_FRONTEND_VERSION}"
 rm -rf ./custom_components/hacs/*.dist-info
 mv ./custom_components/hacs/hacs_frontend ./custom_components/hacs/hacs_frontend_experimental
-
```

Comment:
This should be switched, we want to keep and use `hacs_frontend` but with version  `20240527114629` like in #3727